packages/target-native-android-emulator/src/create-android-emulator-target.js

```diff
@@ -1,9 +1,17 @@
+const fs = require('fs-extra');
+const tempy = require('tempy');
 const osnap = require('osnap/src/android');
 const { createWebsocketTarget } = require('@loki/target-native-core');
 
-const saveScreenshotToFile = filename => osnap.saveToFile({ filename });
+const captureScreenshot = async () => {
+  const filename = tempy.file({ extension: 'png' });
+  await osnap.saveToFile({ filename });
+  const screenshot = await fs.readFile(filename);
+  await fs.unlink(filename);
```

Comment:
What if `fs.readFile` or  `fs.unlink` fail?

Should this be wrapped in a `try-catch` block to handle (e.g. log, report, etc) the error?

Reply:
The try-catch happens on the runner level, if the screenshot fails the story will fail which is correct behavior IMHO. You can't really continue with the comparison after this.

Reply:
If there is already a top-level `try-catch` block, it makes sense :+1: